pygeo/parameterization/DVGeo.py

```diff
@@ -4067,6 +4143,7 @@ def _localDVJacobian(self, config=None):
         variables
         """
 
+        # TODO the comment below is not correct with the addition of shape func DVs. either separate out their implementation or fix the comment.
```

Comment:
Which approach to do think is better? The current implementation reduces code duplication, but separating out the implementation would be more explicit.